modules/twinkleprod.js

```diff
@@ -9,12 +9,14 @@
  *** twinkleprod.js: PROD module
  ****************************************
  * Mode of invocation:     Tab ("PROD")
- * Active on:              Existing articles and files which are not redirects
+ * Active on:              Existing articles, files, books which are not redirects,
+ *                         and user pages not in [[:Category:Wikipedia books (user books)]]
```

Comment:
Its active on user pages _in_ the category.

Reply:
Yes indeed!  Comma-comma-comma confusion.